SEUNAGBEYE/Math-Skill-Game
# #160831436 Add the Star component

### What does this PR do?
Creates the Star component

### Description of Task to be completed?
-add the star component
-add props type validation

## How should this be manually tested?
- Fetch this branch with `git fetch origin ft-star-component-160831436:ft-star-component-160831436`
- Checkout to `ft-star-component-160831436`
- Run `yarn start`
- The app should be started, then visit the webpage and you should see the `Stars Icon`

### Any background context you want to provide?
NA


### What are the relevant pivotal tracker stories?
[#160831436](https://www.pivotaltracker.com/n/projects/2200608/stories/160831436)

### Screenshots (if appropriate)
NA
